lib/commons/color/get-background-color.js

```diff
@@ -65,9 +77,48 @@ function calculateObscuringAlpha(elmIndex, elmStack) {
 }
 
 function elmOutsideParent(elm, bgElm, bgColor) {
-	var visible = (elm !== bgElm && !dom.visuallyContains(elm, bgElm) && bgColor.alpha !== 0);
-	return visible;
+	var outside = (elm !== bgElm && !dom.visuallyContains(elm, bgElm) && bgColor.alpha !== 0);
+	if (outside) {
+		axe.commons.color.incompleteData.set('bgColor', {
+			node: elm,
+			reason: 'elmOutsideParent'
+		});
+	}
+	return outside;
 }
+
+// document.elementsFromPoint misses some elements we need
+// i.e. TR is missing from table elementStack and leaves out bgColor
+// https://github.com/dequelabs/axe-core/issues/273
+function includeMissingElements(elmStack, elm) {
```

Comment:
In theory yes, but in practice the only elements I've seen it miss are `TR`, which can't be positioned; and `LABEL`, but only when it's `display: inline`. The reason I bring this up is I'm not sure how I'd write a test for it given those two scenarios.